packages/front-end/components/Features/EditAttributesModal.tsx

```diff
@@ -126,6 +134,16 @@ export default function EditAttributesModal({ close }: { close: () => void }) {
                     }}
                   />
                 </td>
+                <td>
+                  <Toggle
+                    id={"archived" + i}
+                    label="Archived"
+                    value={!!form.watch(`attributeSchema.${i}.archived`)}
+                    setValue={(value) => {
+                      form.setValue(`attributeSchema.${i}.archived`, value);
+                    }}
+                  />
```

Comment:
I'm slightly worried about making this too easy to toggle in the UI.  I've watched a few people use this form when sharing their screen and their instinct is to turn on all the toggles without reading what they are for.

As an alternative, we could take this action out of the modal entirely.  On the page when we display the table of attributes, we could add a `<MoreMenu>` dropdown for each row with an "Archive" action.  That would do a single API call to archive just that one attribute.

Reply:
This makes sense. An alternate flow would be adding the per-row <MoreMenu> inside the modal itself. To me that also makes sense, since it logically follows from clicking "Edit Attributes". Wdyt?

Reply:
Upon trying them out, I think your first suggestion feels more correct from a UI/UX perspective.